packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -5466,30 +5466,30 @@ curl -v -X DELETE \
 HTTP/1.1 204 No Content
 ```
 
-## Application Logo operations
+## Application logo operations
 
 <ApiLifecycle access="ea" />
 
-### Update Logo for application
+### Update logo for application
 
 <ApiLifecycle access="ea" />
 
 <ApiOperation method="post" url="/api/v1/apps/${applicationId}/logo" />
 
 Update the logo for an application.
 
-> Note: It is only possible to update the logo for applications with a "login" `appLink`.
+> **Note:** You must have a valid login appLinks object to update the logo of an application.
 
-#### Request parameters
+##### Request parameters
 
 | Parameter       | Description                                | Parameter Type   | DataType   | Required |
 | :-------------- | :----------------------------------------- | :--------------- | :--------- | :------- |
 | applicationId   | `id` of an [app](#application-object)      | URL              | String     | TRUE     |
-| file            | file containing logo                       | Body             | File       | TRUE     |
+| file            | File containing logo                       | Body             | File       | TRUE     |
 
-The file must be must be a `PNG`, `JPG`, or `GIF` and less than `1Mb`. For best results use landscape orientation, a transparent background, and a minimum of 420px by 120px to prevent upscaling.
+**The file must be in PNG, JPG, or GIF format, and less than 1 MB in size.** For best results use landscape orientation, a transparent background, and a minimum **size** of 420px by 120px to prevent upscaling.
```

Comment:
Remove the bold from the first sentence and from "size" in the second sentence. We only use bold to denote UI strings.